TypeError in DecomposeMultiDimSqueeze TVM callback in densenet xray variant

## Description

-   TypeError: int() argument must be a string, a bytes-like object or a real number, not 'Any' in DecomposeMultiDimSqueeze TVM callback in densenet xray variant

## Reproduce

```
git checkout kkannan/model_issues
git submodule update --recursive
pytest forge/test/model_demos/high_prio/cnn/pytorch/test_densenet.py -k "xray" -svv
```

## Observed Behaviour
```
>   input_shape = [int(dim) for dim in pre.args[0].checked_type.shape]
E   TypeError: int() argument must be a string, a bytes-like object or a real number, not 'Any'
```

## Logs

- [densenet_xray.log](https://github.com/user-attachments/files/17658982/nov7_densenet_xray.log)

--- a/forge/test/models/pytorch/timeseries/nbeats/test_nbeats.py
+++ b/forge/test/models/pytorch/timeseries/nbeats/test_nbeats.py
@@ -18,7 +18,7 @@ from forge.verify.compare import compare_with_golden
 
 @pytest.mark.nightly
 @pytest.mark.model_analysis
-# @pytest.mark.xfail(reason="Failing with pcc=0.82")
+@pytest.mark.xfail(reason="RuntimeError: Tensor 4 - stride mismatch: expected [24, 1], got [1, 12]")
 def test_nbeats_with_seasonality_basis(test_device):
     compiler_cfg = forge.config._get_global_compiler_config()
 
@@ -46,7 +46,7 @@ def test_nbeats_with_seasonality_basis(test_device):
 
 @pytest.mark.nightly
 @pytest.mark.model_analysis
-# @pytest.mark.xfail(reason="Failing with pcc=0.83")
+@pytest.mark.xfail(reason="Failing with pcc=0.83")
 def test_nbeats_with_generic_basis(test_device):
     compiler_cfg = forge.config._get_global_compiler_config()
 
@@ -68,7 +68,7 @@ def test_nbeats_with_generic_basis(test_device):
 
 @pytest.mark.nightly
 @pytest.mark.model_analysis
-# @pytest.mark.xfail(reason="Failing with pcc=0.83")
+@pytest.mark.xfail(reason="Failing with pcc=0.83")
 def test_nbeats_with_trend_basis(test_device):
     compiler_cfg = forge.config._get_global_compiler_config()
 

